### ISSUE TITLE
Monaco?

### ISSUE THREAD

**Issue by markKL1** [nickgravgaard/AlwaysAlignedVS#14]
Sorry if this is a stupid question, I only know that Monaco is related to VSC; I'm not familiar with how the plugins work for VS / VSC / Monaco.

Would this work out of the box for Monaco? Might one as well start from scratch? Or something inbetween?

**Comment by geroldmeisinger** [comment]
No, but take a look at [Visual Studio Code- Elastic Tabstops Mono](https://marketplace.visualstudio.com/items?itemName=isral.elastic-tabstops-mono)
